Makefile

```
@@ -0,0 +1,186 @@
+# Install required tools
+install-tools:
+	@echo "📦 Installing cargo-generate..."
+	cargo install cargo-generate
+	@echo ""
+	@echo "✅ Tools installed successfully!"
+	@echo ""
+	@echo "You can now run:"
+	@echo "  make test-full"
+
+# Clean up test directories
+clean:
```

checkmake (0.2.2)

[warning] Target body for "clean" exceeds allowed length of 5 (7).

(maxbodylength)